### PR TITLE
fix: resolved import errors `ArgillaTrainer`

### LINKED ISSUE #3345: [BUG] `ArgillaTrainer` imports are not working correctly
**Describe the bug**
Import for the `ArgillaTrainer` w.r.t. TaskMapping are incorrect.

**Stacktrace and Code to create the bug**
N.A.

**Expected behavior**

**Environment:**
 - Argilla Version [e.g. 1.0.0]: 1.12
 - ElasticSearch Version [e.g. 7.10.2]:
 - Docker Image (optional) [e.g. argilla:v1.0.0]:

**Additional context**
N.A.

### PR DESCRIPTION
**Pull request by davidberenstein1957**
# Description

The imports for `ArgillaTrainer` where not setup correctly.

Closes #3345 

**Type of change**

- [X] Bug fix (non-breaking change which fixes an issue)

**How Has This Been Tested**

N.A.

**Checklist**

- [X] I added relevant documentation
- [X] follows the style guidelines of this project
- [X] I did a self-review of my code
- [X] I made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: docs/_source/guides/llms/practical_guides/fine_tune_others.md [hunk 1]
```diff
@@ -61,12 +61,12 @@ task_mapping = TrainingTaskMapping.for_text_classification(
 
 :::{tab-item} LabelQuestion
 ```python
-from argilla import FeedbackDataset, TrainingTaskMapping
+import argilla.feedback as rg
```
Comment (frascuchon) [inline docs/_source/guides/llms/practical_guides/fine_tune_others.md]: Do we want to import this with the same alias as `import argilla as rg`?

I think can be confusing for user when they want mix feedback trainings opérations with other ones.

I would say just import required classes normally

```
from argilla.feedback import FeedbackDataset, TrainingTaskMapping
```

Comment (davidberenstein1957) [reply docs/_source/guides/llms/practical_guides/fine_tune_others.md]: @frascuchon, I changed this.

File: docs/_source/guides/llms/practical_guides/fine_tune_others.md [hunk 2]
```diff
@@ -140,15 +140,22 @@ dataset.prepare_for_training(
...
-dataset = rg.FeedbackDataset(
+dataset = FeedbackDataset(
     guidelines="Add some guidelines for the annotation team here.",
     fields=[
-        rg.TextField(name="text", title="Human prompt"),
+        TextField(name="text", title="Human prompt"),
     ],
     questions =[
-        rg.LabelQuestion(
+        rLabelQuestion(
```
Comment (frascuchon) [inline docs/_source/guides/llms/practical_guides/fine_tune_others.md]:
```suggestion
        LabelQuestion(
```

**Review by frascuchon — APPROVED** [review]
Just a typo fix in docs

**Pull request merged**